[IMP] accounting: tax return (#2909)
odoo/documentation: content/applications/finance/accounting/reporting/declarations/tax_returns.rst

Tax return (VAT declaration)
============================

> Companies with a registered for :abbr:`VAT (Value Added Tax)` number must submit a **tax return** on

Reviewer: ```suggestion
Companies with a registered :abbr:`VAT (Value Added Tax)` number must submit a **tax return** on
```

a monthly or quarterly basis, depending on their turnover and the registration regulation. A tax
return - or VAT return - gives the tax authorities information about the taxable transactions made
by the company. The **output tax** is charged on the number of goods and services sold by a
business, while the **input tax** is the tax added to the price when goods or services are

> purchased.
> Based on these values, the company can calculate the tax amount they have to pay or be refunded.

Reviewer: going to the next line is the same as a space in RST.
```suggestion
purchased. Based on these values, the company can calculate the tax amount they have to pay or be refunded.
```

.. note::
   You can find additional information about VAT and its mechanism on this page from the European

>    `"What is VAT?" Commission <https://ec.europa.eu/taxation_customs/business/vat/what-is-vat_en>`_.

Reviewer: problem with the text and link here

Prerequisites
=============

Tax Return Periodicity
----------------------

The configuration of the **Tax Return Periodicity** allows Odoo to compute your tax return correctly
and also to send you a reminder to never miss a tax return deadline.

> To do so, go to :menuselection:`Accounting --> Configuration --> Settings`.Under the

Reviewer: ```suggestion
To do so, go to :menuselection:`Accounting --> Configuration --> Settings`. Under the
```

- :guilabel:`Periodicity`: define here whether you submit your tax return on a monthly or quarterly
  basis;

> 

Reviewer 2: ```suggestion

```
Gap between the lines

Tax Grids
---------

Odoo generates tax reports based on the :guilabel:`Tax Grids` settings that are configured on your
taxes. Therefore, it is crucial to make sure that all recorded transactions use the right taxes.

> You can see the :guilabel:`Tax Grids` by opening any :guilabel:`Invoice` and :guilabel:`Bill` in the

Reviewer 2: ```suggestion
You can see the :guilabel:`Tax Grids` by opening the :guilabel:`Journal Item tab` of any :guilabel:`Invoice` and :guilabel:`Bill`.
```

Reviewer 2: The previous order was confusing/misleading

> :guilabel:`Journal Item tab`.

Reviewer 2: ```suggestion

```

   :alt: Configure taxes and their tax grids in Odoo Accounting

.. note::
   Taxes and reports are usually already pre-configured: a *Fiscal Localization Package* is
   installed according to the country you select at the creation of your database. :doc:`Click here
   <../../fiscal_localizations/overview/fiscal_localization_packages>` for more information about
   Fiscal Localization Packages.

> .. _tax_return_lock:

Reviewer: why did you delete this anchor?

Author: I didn't see the point of it, in this specific point.

>    for more information about Fiscal Localization Packages.

Reviewer: you can put a link directly on "fiscal localization package" and write this in sentence case.

a report once its period is closed.

Therefore, we recommend locking your tax date before working on your
:guilabel:`Closing Journal Entry`.
This way, other users cannot modify or add transactions that would have an impact on the
:guilabel:`Closing Journal Entry`, which can help you avoid some tax declaration errors.

To check the current :guilabel:`Tax Lock Date`, or to edit it, go to
:menuselection:`Accounting --> Accounting --> Actions: Lock Dates`.

.. image:: tax_returns/tax_return_lock.png
   :align: center
   :alt: Lock your tax for a specific period in Odoo Accounting

Tax Report
----------

Once all the transactions involving taxes have been posted for the period you want to report, open
your :guilabel:`Tax Report` by going to :menuselection:`Accounting --> Reporting -->
Audit Reports: Tax Report`.
Make sure to select the right period you want to declare by using the date filter. You can see an
overview of your tax report.
From this view, you can easily access different formats of your tax report, such as `PDF` and
XLSX.
These include all the values to report to the tax authorities, along with the amount you have to pay

> or be refunded.

Reviewer: going to the next line doesn't work in RST (also, it's not something usual in English)
either write a space or make it a new paragraph.

Tax Lock Date
-------------

> Any new transaction whose accounting date is before to the :guilabel:`Tax Lock Date` has its tax

Reviewer: marked as resolved but it's still there.
Isn't the previous "prior to" actually better?
